en/docs/wip/change-developer-portal-password.md

```diff
@@ -1,64 +1,95 @@
-
-Follow the instructions below to change your password: 
-
-1. Sign in to the Developer Portal.    
-
-     `https://<hostname>:9443/devportal`     
-
-     Example: `https://localhost:9443/devportal`    
-
-     Use your username and password to sign in.    
-
-2. Click  on your username at the top right corner and select  **Change Password**.  
-
-     [![Change Developer Portal password User Menu]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)    
-
-3. Enter your old password, a new password, re-enter the new password, and thereafter click **SAVE** to submit changes.
-
-     [![Change Developer Portal password Save Clicking]({{base_path}}/assets/img/learn/change-devportal-password-policy-guideline-display.png)]({{base_path}}/assets/img/learn/change-devportal-password-policy-guideline-display.png)
+# Change the Developer Portal Password  
+  
+  You can change your own password for the WSO2 API Manager Developer Portal using the Developer Portal UI.    
+  
+Follow the instructions below to change your password:   
+  
+1. Sign in to the Developer Portal.      
+  
+     `https://<hostname>:9443/devportal`   
+  Example: `https://localhost:9443/devportal`   
+  Use your username and password to sign in.      
+  
+2. Click  on your username at the top right corner and select  **Change Password**.    
+  
+     [![Change Developer Portal password User Menu]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)]({{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png)      
+  
+3. Enter your old password, a new password, re-enter the new password, and thereafter click **SAVE** to submit changes. The new password should adhere to the custom password policies as described below.
+  
+     [![Developer portal password change submit]({{base_path}}/assets/img/learn/change-devportal-password-submiting.png)]({{base_path}}/assets/img/learn/change-devportal-password-submiting.png)  
+  
+## Use custom password policy
+You can define your custom password policy by defining one or both of the followings. 
+    
+1. [User store password regex](https://apim.docs.wso2.com/en/latest/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-jdbc-user-store/#configuring-a-jdbc-user-store)  
+Example:
+```toml
+[user_store]
+type = "database_unique_id"
+
+[user_store.properties]
+PasswordJavaRegEx = "^[\\S]{6,30}$"
+```
+2. [Identity management  password policies](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/identity-management-for-the-api-dev-portal/#password-policies)
+
+<html>    
+    <div class="admonition note">    
+        <p class="admonition-title">Note</p>    
+        <p>
+            The password policy set by identity management password policies can be overridden by Management console for each tenant.
```

Comment:
Shall we change the above line as this. "The password policy set by identity management password policies through the configuration file can be overridden by Management console for each tenant."